FlowCrypt/Controllers/Setup/SetupGenerateKeyViewController.swift

```diff
@@ -89,7 +91,7 @@ final class SetupGenerateKeyViewController: SetupCreatePassphraseAbstractViewCon
     private func putKeypairsInEncryptedStorage(encryptedPrv: CoreRes.GenerateKey, storageMethod: StorageMethod, passPhrase: String) throws {
         try appContext.encryptedStorage.putKeypairs(
             keyDetails: [encryptedPrv.key],
-            passPhrase: storageMethod == .persistent ? passPhrase: nil,
+            passPhrase: storageMethod == .persistent || clientConfiguration.shouldRememberPassphraseByDefault ? passPhrase: nil,
```

Comment:
Same here. Plus, in situations when user actually has an option to choose (like when creating a key, there is radio select about method of storing pass phrase) you should not give user options that will then be ignored.

Also, I have misled you somewhat.

See https://flowcrypt.com/docs/business/org-rules.html

`DEFAULT_REMEMBER_PASS_PHRASE` means `Changes a setup checkbox default, in a way that browser will remember user pass phrase` which I'll rephrase more precisely as `Sets persistently storing pass phrase as the default choice in setup UI.`

Given there is no user UI involved regarding this in the EKM setup flow, `DEFAULT_REMEMBER_PASS_PHRASE` is not the right choice.

Simply removing flag `FORBID_STORING_PASS_PHRASE` should achieve the desired result (for EKM flow, at least). And if it doesn't, that's the fix that should be changed.

I'll close this PR and file a more appropriate issue.